docs/adpod_setup.md

```diff
@@ -17,6 +17,8 @@
 |`VIDEO_LENGTHS`| Adpod video creative durations.<br> Ex `VIDEO_LENGTHS = [5,10,15]` will create creatives  with durations 5, 10 and 15 secs for each slot of Adpod. | integer array|
 |`PREBID_BIDDER_CODE` | Bidder codes to target bidders with one line item. <br>Ex `PREBID_BIDDER_CODE = ['pubmatic']`. This parameter is mandatory for bidder level reporting. Set to `None` to generate line items for all partners.   | string array|
 |`OPENWRAP_BUCKET_CSV` | Set this to one of the csv file mentioned in ` Inline Header Bidding  csv` table  below. This CSV lists buckets and price granularity; it sets `pwtpb` targeting for each line item..| string |
+|`ENABLE_DEAL_LINEITEM` | Set this value to `TRUE` for creating deal line items| boolean |
+|`DEAL_CONFIG` | Configuration for creating deal lineitem. Set this if `ENABLE_DEAL_LINEITEM` is set to `TRUE`. `DEAL_CONFIG` should have price, array of prefix and array of mindealtier values. Ex: `{"pubmatic":{"price":10,"prefix":["abc"],"mindealtier":[5]}}` | object |
```

Comment:
Also add what would be the currency here? Is it USD default?

Reply:
Updated the readme.